manifests-gen/customizations.go

```diff
@@ -326,3 +334,59 @@ func mergeMaps[K comparable, V any](maps ...map[K]V) map[K]V {
 	}
 	return result
 }
+
+// addInfraClusterProtectionPolicy adds a Validating Admission Policy and Binding for protecting
+// InfraClusters created by the cluster-capi-operator from deletion and editing.
+func addInfraClusterProtectionPolicy(objs []unstructured.Unstructured, providerName string) []unstructured.Unstructured {
+	policy := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "admissionregistration.k8s.io/v1beta1",
+			"kind":       "ValidatingAdmissionPolicy",
+			"metadata": map[string]interface{}{
+				"name": "protect-capi-" + providerName + "cluster",
```

Comment:
Can we name these similarly to how I'm naming the cloud controller manager ones, so, perhaps
```
openshift-cluster-api-protect-<infra>-cluster
```
Want to make it obvious that this is a core openshift thing and which namespace it belongs to

Reply:
Sounds good. Done